### ISSUE TITLE
no field `gpio25`

### ISSUE THREAD

**Issue by blippy** [thejpster/pico-blink-rs#3]
Issuing
`cargo build`

results in the output:
```
  Compiling pico-blink-rs v0.2.0 (/home/pi/pico-blink-rs)
error[E0609]: no field `gpio25` on type `PADS_BANK0`
   --> src/main.rs:126:18
    |
126 |     p.PADS_BANK0.gpio25.write(|w| {
    |                  ^^^^^^ unknown field

error[E0609]: no field `gpio25_ctrl` on type `IO_BANK0`
   --> src/main.rs:133:16
    |
133 |     p.IO_BANK0.gpio25_ctrl.write(|w| {
    |                ^^^^^^^^^^^ unknown field

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0609`.
error: could not compile `pico-blink-rs`

To learn more, run the command again with --verbose.

```

**Comment by gambrose** [comment]
I just encountered this too.
It appears that it is fixed by #2 👍

**Comment by 9names** [comment]
#2 is merged now, thank you for your patience!

Just a note that this project is an example of how to use the PAC directly to interact with hardware.
We have HAL-based blinky examples that are a better place to start if you're looking to work with the rest of the rust embedded ecosystem.
https://github.com/rp-rs/rp-hal/blob/main/rp2040-hal/examples/blinky.rs
https://github.com/rp-rs/rp-hal/blob/main/boards/pico/examples/pico_blinky.rs
